src/AspNet.Security.OAuth.Introspection/Events/IntrospectTokenContext.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json.Linq;
+
+namespace AspNet.Security.OAuth.Introspection {
+    public class IntrospectTokenContext : BaseIntrospectionContext {
```

Comment:
`BeSoKindAndIntrospectThisTokenContext.cs` would be a polite and clear way of saying what this has to do.. :smile:
Hmm... `IntrospectIncomingToken`, sth like that?..
When you understand the intention of this event, the name looks clear. But when you see it for the first time - I got confused, so can other people.